tests/storage/blockvolume_test.py

```diff
@@ -44,16 +44,49 @@
 from testlib import permutations, expandPermutations
 from testlib import VdsmTestCase
 
-
+CHUNK_SIZE_MB = 1024
 CONFIG = make_config([
-    ('irs', 'volume_utilization_chunk_mb', '1024'),
+    ('irs', 'volume_utilization_chunk_mb', str(CHUNK_SIZE_MB)),
     ('irs', 'volume_utilizzation_precent', '50'),
 ])
 
 
+@contextmanager
+def make_volume(size, format=sc.RAW_FORMAT, prealloc=sc.SPARSE_VOL):
+    img_id = make_uuid()
+    vol_id = make_uuid()
```

Comment:
Maybe better use `img_uuid` & `vol_uuid`?

Reply:
ok, this is actually moved code, not created new, so I moved it as is. But it is ok, I can update the name of these variables in the process.